Improve key rotation section

andymunro opened the pull request:
Closes #19166

This replaces https://github.com/keycloak/keycloak-documentation/pull/1750 which had to moved to the new repository.

@ikhomyn Can you approve this again and @abstractj can you check if this works.  I changed the KeyPair terms and removed one instance of wildfly.

andymunro committed:
Improve key rotation section

Closes #19166

ikhomyn commented on docs/documentation/server_admin/topics/realms/keys.adoc:
```diff
@@ -22,35 +22,78 @@ that is able to provide an active keypair.
++
+[Source,Bash,Subs=+Attributes]
+----
+keytool -importcert -file domain.crt -keystore <keystore>.jks -storepass <password>  -alias <key>
```
```suggestion
keytool -importcert -file domain.crt -keystore <keystore>.jks -storepass <password> -alias <key>
```

ikhomyn reviewed (approved):
Hi @andymunro, looks good

andymunro commented:
@ikhomyn Can you approve this again and @abstractj can you check if this works. I changed the KeyPair terms and removed one instance of wildfly.

ikhomyn reviewed (dismissed):
Hi, @andymunro Looks good.

abstractj committed:
Update the usage of the word key pair

In the context of this document, key pair is referring to a public and private key, instead of a Java class.

abstractj commented:
@andymunro submitted my suggested changes on top of yours. Could you please review?

andymunro reviewed (commented):
looks good
